### DISCUSSION TITLE
[Feature] Installer (alternative to Microsoft Store)

### DISCUSSION THREAD

**Discussion by Hofknecht** [Hofknecht/SystemTrayMenu#408] [Ideas]
in https://github.com/Hofknecht/SystemTrayMenu/discussions/316 we need an installer
and also for autoupdater we need an installer https://github.com/Hofknecht/SystemTrayMenu/issues/185

maybe this visual studio installer is something we should try

(currently you can only use as installler the version from Microsoft Store)

**Comment by Tanjalibertatis** [comment]
Create an installer project in VS with this extension: https://marketplace.visualstudio.com/items?itemName=VisualStudioClient.MicrosoftVisualStudio2022InstallerProjects&ssr=false#overview

**Comment by Hofknecht** [comment]
currently not planned, let's convert into discussion
